bazel/pbgen/main.go

```diff
@@ -985,10 +979,8 @@ func (g *implGenerator) generateMapFieldWriteJson(f protoreflect.FieldDescriptor
 		w.Println("w.string(value);")
 	case protoreflect.MessageKind:
 		switch {
-		case f.MapValue().Message().FullName() == "google.protobuf.Duration":
-			w.Printf("w.append_raw_json(serde::pb::json::wellknown::duration_to_json(value));\n")
-		case f.MapValue().Message().FullName() == "google.protobuf.FieldMask":
-			w.Printf("w.append_raw_json(serde::pb::json::wellknown::field_mask_to_json(value));\n")
+		case isWellKnownType(f.MapValue().Message()):
+			w.Printf("w.append_raw_json(serde::pb::json::wellknown::%s_to_json(value));\n", pascalToSnakeCase(string(f.MapValue().Message().Name())))
```

Comment:
Decided to make it easier to extend in the future and just codify the existing convension